### PR TITLE
trade

### PR DESCRIPTION
**Pull request by martinupset**
will fix the conflict

### CODE REVIEW

**Comment by martinupset** [comment]
this is the trade page

File: frontend-opal/src/reducers/trade.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const pairs = {
+  CHANGE_NAME: (state, action) => ({
```
Comment (adamgyulavari) [inline frontend-opal/src/reducers/trade.js]: @martinupset `CHANGE_CURRENT_TRADE_ITEM`

File: frontend-opal/src/thunks/showtrade.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import actions from 'actions/showTrade';
+
+function showTradeName(input) {
```
Comment (adamgyulavari) [inline frontend-opal/src/thunks/showtrade.js]: @martinupset this is not needed to be a thunk just create a simple action

File: frontend-opal/src/components/WatchList/InstrumentRow/InstrumentRow.js [hunk 3]
```diff
@@ -26,6 +27,10 @@ function InstrumentRow({ data }) {
           iconStyle={styles.icon}
           iconSource={transIcon}
           blockStyle={styles.iconContainer}
+          func={() => {
```
Comment (adamgyulavari) [inline frontend-opal/src/components/WatchList/InstrumentRow/InstrumentRow.js]: @martinupset func?

File: frontend-opal/src/thunks/showtrade.js (hunk 2 above)
Comment (martinupset) [reply frontend-opal/src/thunks/showtrade.js]: ok

File: frontend-opal/src/reducers/trade.js (hunk 1 above)
Comment (martinupset) [reply frontend-opal/src/reducers/trade.js]: ok

File: frontend-opal/src/components/WatchList/InstrumentRow/InstrumentRow.js (hunk 3 above)
Comment (martinupset) [reply frontend-opal/src/components/WatchList/InstrumentRow/InstrumentRow.js]: already done, can you recheck it?

**Comment by martinupset** [comment]
all done

**Pull request merged**